src/context.rs

```diff
@@ -4140,16 +4084,14 @@ impl<'a> ContextWriter<'a> {
       }
 
       if level > NUM_BASE_LEVELS as u32 {
```

Comment:
level > ... as u32 then convert level to u16. This has code smell.

Reply:
I think NUM_BASE_LEVELS was a #define in libaom so changing its type to u16 would also be a legit fix

Reply:
actually no we actually do the conversion at v.abs() as u16 right above. I think we should just drop all those conversions